ParisBNF/abb/BNFabb157.xml

```diff
@@ -0,0 +1,488 @@
+                          <locus from="13ra" to="24vb"/>
+                         <locus from="36ra" to="36vb"/>
+                          <title type="complete">About the Tabernacle</title>
+                            <textLang mainLang="gez" otherLangs="am"/>
+                       <incipit xml:lang="gez"><hi rend="rubric">ንወጥን፡ በረድኤተ፡ እግዚአብሔር፡ ጽሒፈ፡ ግብረ፡ ደብተራ፡</hi>ዘኦሪት፡ አዕናቅ፡
+                         ኢያስጲስ፡ ወአቡቀለምሲስ፡ ያስቢስ፡ ወጥልም፡ በኦሪት። 
+                       </incipit>
+                            </msItem>
+
+                            <msItem xml:id="p1_i1.3">
+                              <locus from="36vb" to="39vb"/>
+                              <title type="complete" ref="LIT4117Interpretation#Leviticus">Commentary on Leviticus</title>
+                              <textLang mainLang="gez" otherLangs="am"/>
+                              <incipit xml:lang="gez">፫፡ ወ፴ወ፩፡ ዓመት፡ እምአመ፡ ወፅኡ፡ ፳ኤል፡ እምግብጽ፡ ተጽሕፈ፡ ኦሪት፡ ዘሌዋውያን፡ ዝውእቱ፡ ፍካሬሁ፡ ቃሉ፡ ክዱን፡ 
+                                መባአ፡ ዘ፡ ዘያበውኡ፡ እስራኤል፡ ለእግዚአብሔር፡ ዘንቱ፡ ውእቱ፡ ላህም፡ ወ በግዕ፡ ፍየል፡ ወርግብ፡ ወማዕነቅ።  ማዕነቅ፡ ዘ፡ ዋኖስ። ወእምነ፡ እክል፡ ስንዴ፡ ዕጣነ፡ ወቅብዕ።
+                                ወባዕደሰ፡ ኢያበውዑ፡ ለመሥዋዕት፡ 
+                              </incipit>
+                            </msItem>
+                                  
+                              <msItem xml:id="p1_i1.4">
+                                <locus from="39vb" to="43rb"/>
+                                <title type="complete" ref="LIT4117Interpretation#Numbers">Commentary on Numbers</title>
+                                <textLang mainLang="gez" otherLangs="am"/>
+                                <incipit xml:lang="gez">በ፴፡ ወ፰፡ ዓመት፡ እምአመ፡ ወፅኡ፡ እስራኤል። ተጽሕፈ፡ ኦሪት፡ ዘኍልቍ፡ ኍልቍሰ፡ ዘተብህለ፡ እስመ፡ ይነግር፡ ኆልቆሙ፡ ለእስራኤል፡ ዛቲ፡ ይእቲ፡ 
+                                  ፍቅዶ፡ ዘአስተፋቀድዎሙ፡ ሙሴ፡ ወአሮን፡ ወመላእክት። ወኮነ፡ ፍ<pb n="40ra"/>ቅዶሙ፡ ለደቂቀ፡ እስራኤል፡ ዘእም፳፡ ዓም፡ ወለእለ፡ እምኔሁ። ኵሉ፡ ዘይጽብእ፡ ፀብአ፡  
+                                    </incipit>
+                              </msItem>  
+                                  
+                              <msItem xml:id="p1_i1.5">
+                                <locus from="43rb" to="45va"/>
+                                <title type="complete" ref="LIT4117Interpretation#Deuteronomy">Commentary on Deuteronomy</title>
+                                <textLang mainLang="gez" otherLangs="am"/>
+                                <incipit xml:lang="gez" >በ፵፡ ዓመት፡ እምአመ፡ ወፅኡ፡ እስራኤል፡ እምግብጽ፡ ተጽሕፈ፡ ኦሪት፡ ዘዳግም፡ ዝውእቱ፡ ትርጓሜ፡ ሕቱም፡ ጻረባ፡ በዳ፡ ፋራን፡ ዘ፡ ዝኒ፡ በዳ፡ ኮሬብ፡ ዘ፡ ዝወው፡ ሌባ፡ ዘ፡ ቀኝእ፡ 
+                                  ጳራልያ፡ ጽንፈ፡ ባሕር፡ አንጢሊባኖስ፡ ዘ፡ አንጻረ፡ ሊባኖስ፡  
+                                </incipit>
+                              </msItem>
+                              
+                              <msItem xml:id="p1_i1.6">
+                                <locus from="45va" to="47rb"/>
+                                <title type="complete" ref="LIT4117Interpretation#Joshua">Commentary on Joshua</title>
```

Comment:
is there a DIV/ID for Joshua in `LIT4117`?

Reply:
yes, it should be there